index_test.go

```diff
@@ -1115,7 +1115,7 @@ func TestListRepos(t *testing.T) {
 			Stats: RepoStats{
 				Shards:                     1,
 				Documents:                  4,
-				IndexBytes:                 308,
+				IndexBytes:                 300,
```

Comment:
We could instead udpate `memoryUse` like this
```diff
- sz += 8 * len(d.runeDocSections)
+ sz += 8 * len(d.runeDocSectionsRaw)
```

Reply:
if its the raw section then it is mmap and isn't part of our stable memory usage.